app/models/reports/repository.rb

```diff
@@ -209,26 +232,28 @@ def initialize(regions, periods:)
       }
     end
 
-    # This method currently always returns the "excluding LTFU denominator".
-    # Repository only returns "excluding LTFU" rates.
     # This only powers queries for children regions which do not require both variants of control rates, unlike Result.
     # As we deprecate Result and shift to Repository, a Repository object should be able to return both rates.
-    def denominator(region, period)
-      cumulative_assigned_patients_count[region.slug][period.adjusted_period] - ltfu_counts[region.slug][period]
+    def denominator(region, period, with_ltfu: false)
+      if with_ltfu
+        cumulative_assigned_patients_count[region.slug][period.adjusted_period]
+      else
+        cumulative_assigned_patients_count[region.slug][period.adjusted_period] - ltfu_counts[region.slug][period]
+      end
```

Comment:
Needed this method to be able to return the patient count `with_ltfu` included as well, to support the toggle in the reports.